src/models/installation.ts

```diff
@@ -31,11 +34,31 @@ export default class Installation extends Sequelize.Model {
 	}
 
 	static async getForHost(host: string): Promise<Installation | null> {
-		return Installation.findOne({
+		const payload: any = {
 			where: {
-				jiraHost: host
+				jiraHost: host,
 			}
-		});
+		}
+
+		if (await sortInstallationsByIdFlagIsOn(host)) {
+			payload.order = [["id", "DESC"]]
+		}
+
+		return Installation.findOne(payload);
+	}
+
+	static async getAllForHost(host: string): Promise<Installation[]> {
+		const payload: any = {
+			where: {
+				jiraHost: host,
+			}
+		}
+
+		if (await sortInstallationsByIdFlagIsOn(host)) {
+			payload.order = [["id", "DESC"]]
+		}
```

Comment:
/super-nit if it is a new function which never used (and we don't use it anywhere yet) we don't need to use a feature flag here. (Also ordering doesn't matter that much when we retrieve all elements anyway.

Reply:
Just trying to play it ultra safe. Christian told me, until the app is stable, all changes should have tests and FFs. I agree, it's probably overkill here but easy to clean up.